### ISSUE TITLE
Push notification (silent or visible) not received for CanBikeCO

### ISSUE THREAD

**Issue by shankari** [e-mission/e-mission-docs#749]
While testing https://github.com/e-mission/e-mission-docs/issues/735, we found that:
- @asiripanich installed CanBikeCO but did not receive any silent push notifications (https://github.com/e-mission/e-mission-docs/issues/735#issuecomment-1175766262)
- @asiripanich also did not receive any visible push notifications (https://github.com/e-mission/e-mission-docs/issues/735#issuecomment-1175772822)
- the token in the uploaded logs and the profile DB were different https://github.com/e-mission/e-mission-docs/issues/735#issuecomment-1175774633

We are now trying to detemine why the tokens are different.

**Comment by shankari** [comment]
> Can you explain a bit more? why do they need to overlap? I installed them at different times and only one app was installed each time.

- the push notification is likely failing because of a mismatch between the push token in the profile and the push token in the logs
- the push token was updated in the profile at `2022-07-06 04:30:43`
- it would be good to look at the logs around `2022-07-06 04:30:43` to see the token that was generated on the phone, and whether it was updated correctly in the server profile.

All the logs that you uploaded were before `2022-07-06 00:00:00` so do not indicate which token was sent to the server.

**Comment by asiripanich** [comment]
I don't think this is specific to CanBikeCO.. I tested OpenPATH yesterday and got the same issue.

**Comment by shankari** [comment]
Well, I know that the push notification worked for @sebastianbarry on OpenPATH.
https://github.com/e-mission/e-mission-docs/issues/735#issuecomment-1175307154

And I don't have logs from OpenPATH showing that the push notifications **don't work**. 

Yesterday, you got OpenPATH into the START state after install, but I am not sure whether you waited for an hour to see if the push notification fixed it.

So I've marked this for CanBikeCO for now. If I can confirm that OpenPATH also does not receive push notifications, I can edit the issue title.
